### DISCUSSION TITLE
Support for BBEdit ??

### DISCUSSION THREAD

**Discussion by sscotti** [community/community#52231] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I just signed up for Co-Pilot trial.  I have been a "Mac" user for decades and have also been using BBEdit as a Text Editor for about the same amount of time.  I do have Visual Studio Code, and that is in many ways superior to BBEdit when it comes to the variety of extensions available.

Just wondering if there are any plans to add BBEdit support within Co-Pilot or should I contact the vendor, Bare Bones Software ?

**Comment by daBee** [comment]
Get into their Google group.  Or email them.  I'd like to see this as well.
